test/DynamoCoreWpfTests/PackageManagerUITests.cs

```diff
@@ -91,6 +95,39 @@ public void PackagePublishWindowClosesWithDynamo()
 
         }
 
+        [Test]
+        public void AdddingaPackageRaisesCanExecuteChangeOnDelegateCommand()
+        {
+            //create a new synchronization context since the package searchview from previous UI tests is still
+            //running and throws exception when attempting to modify searchresults
+            SynchronizationContext.SetSynchronizationContext(new SynchronizationContext());
+            var vm = new PublishPackageViewModel(ViewModel);
+            ViewModel.OnRequestPackagePublishDialog(vm);
```

Comment:
@pboyer to clarify: you mean use the current sync context, correct? If you mean the publishPackageViewModel, I'm not sure where one exits already?